### PR TITLE
feat: created components, added homepage layout

### PR DESCRIPTION
**Pull request by a-popova**
Where is it better to store file with mixins? (colors, for example)

### CODE REVIEW

File: src/components/app-header/app-header.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React, {Component} from 'react';
+import './app-header.scss';
+
+export default class AppHeader extends Component {
```
Comment (DashaKavalenka) [inline src/components/app-header/app-header.js:4]: use functional component

File: src/components/app-header/app-header.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
+$header-black: #242424;
+$gray: #989A9A;
+$dark-gray: #5C5D5D;
+$blue: #4BB9F9;
+$light-gray: #F0F3F4;
```
Comment (DashaKavalenka) [inline src/components/app-header/app-header.scss:5]: Move it to variable file

File: src/components/app/app.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React, {Component} from 'react';
+import './app.scss';
+
+import AppHeader from '../app-header';
+import AppFooter from '../app-footer';
+import Home from '../home';
+
+export default class App extends Component {
```
Comment (DashaKavalenka) [inline src/components/app/app.js:8]: Use functional component

File: src/components/home/home.scss [hunk 4]
```diff
@@ -0,0 +1,54 @@
+$header-black: #242424;
+$gray: #989A9A;
+$dark-gray: #5C5D5D;
+$blue: #4BB9F9;
+$light-gray: #F0F3F4;
```
Comment (DashaKavalenka) [inline src/components/home/home.scss:5]: Move it and import here

**Comment by a-popova** [comment]
Made changes according to your comments. Please, check it

**Pull request merged**